tests/integration/test_modular_decomposition_isomorphisms.py

```diff
@@ -0,0 +1,21 @@
+import unittest
+
+from color_refinement_helper import graph_to_modules, modules_to_graph_with_module_isomorphism
+from tools import create_graph_helper
+
+
+class Test(unittest.TestCase):
+    def test_moeilijkere_graaf(self):
+        g = create_graph_helper(
+            [(0, 1), (1, 2), (2, 3), (3, 4), (4, 5), (5, 6), (6, 7), (5, 7), (5, 8), (3, 8), (3, 9), (1, 9), (2, 9),
+             (1, 10), (6, 11), (7, 11), (11, 12), (11, 13), (11, 14), (12, 13), (12, 14), (13, 14)])
+
+        h = g.deepcopy()
+        md = graph_to_modules(g + h)
+        graph_md, modular_iso = modules_to_graph_with_module_isomorphism(md)
```

Comment:
Modules zitten nu 2x in de `modular_iso`, dat is vast niet de bedoeling.
Eg. hier geeft het `[0+10 , 4+8, 0+10 , 4+8], [2+9, 6+7, 2+9, 6+7], [12+13+14, 12+13+14]`

Reply:
Lol laat maar, ik had de 'g + h' over het hoofd gezien.

Reply:
Inderdaad, dat is precies de bedoeling.